python/nvtx/nvtx.py

```diff
@@ -231,3 +233,28 @@ def end_range(range_id):
         The `RangeId` object returned by the `start_range` function.
     """
     libnvtx_end_range(range_id)
+
+
+def enabled():
+    """
+    Returns True if nvtx is enabled.
+    """
+    return not os.getenv("NVTX_DISABLE", False)
```

Comment:
If we want this to be part of the public API we likely need to do something like storing the result at runtime in `_enabled` and then return that value in future invocations.

Reply:
May also want to rename this to `is_enabled()` just to make it clear that this is a check and not something to call to change a setting.

Reply:
👍 fixed